static/app/utils/metrics/dashboard.tsx

```diff
@@ -34,24 +35,26 @@ export function toDisplayType(displayType: unknown): DisplayType {
   return DisplayType.LINE;
 }
 
-export function defaultMetricWidget(selection: PageFilters) {
-  return convertToDashboardWidget(
-    [{...selection, ...emptyMetricsQueryWidget}],
-    MetricDisplayType.LINE
-  );
-}
-
-export function getWidgetQuery(metricsQuery: MetricsQuery) {
+export function getWidgetQuery(metricsQuery: MetricsQuery & {id?: number}): WidgetQuery {
   const field = MRIToField(metricsQuery.mri, metricsQuery.op);
-
   return {
-    name: '',
+    name: defined(metricsQuery.id) ? `${metricsQuery.id}` : '',
     aggregates: [field],
     columns: metricsQuery.groupBy ?? [],
     fields: [field],
     conditions: metricsQuery.query ?? '',
-    // @ts-expect-error TODO: change type
-    orderby: metricsQuery.orderBy,
+    orderby: 'desc',
```

Comment:
this should not be an issue since there are no tables in metrics but orderby needs to be undefined by default for table widgets